main.go

```diff
@@ -1,63 +1,95 @@
 package main
 
 import (
+	"context"
 	"github.com/CMedrado/DesafioStone/domain/account"
 	"github.com/CMedrado/DesafioStone/domain/authentication"
 	"github.com/CMedrado/DesafioStone/domain/transfer"
-	https "github.com/CMedrado/DesafioStone/https"
+	"github.com/CMedrado/DesafioStone/https"
 	http_account "github.com/CMedrado/DesafioStone/https/account"
 	http_login "github.com/CMedrado/DesafioStone/https/authentication"
 	http_transfer "github.com/CMedrado/DesafioStone/https/transfer"
-	store_account "github.com/CMedrado/DesafioStone/storage/file/account"
-	store_token "github.com/CMedrado/DesafioStone/storage/file/token"
-	store_transfer "github.com/CMedrado/DesafioStone/storage/file/transfer"
+	"github.com/CMedrado/DesafioStone/storage/postgre"
+	store_account "github.com/CMedrado/DesafioStone/storage/postgre/accounts"
+	store_token "github.com/CMedrado/DesafioStone/storage/postgre/token"
+	store_transfer "github.com/CMedrado/DesafioStone/storage/postgre/transfer"
+	"github.com/jackc/pgx/v4/pgxpool"
 	"github.com/sirupsen/logrus"
 	"net/http"
 	"os"
-	time "time"
+	"time"
 )
 
 const dbFileNameAccount = "accounts.db.json"
 const dbFileNameToken = "token.db.json"
 const dbFileNameTransfer = "transfer.db.json"
 const Port = "5000"
+const dbUrl = "postgresql://postgres:example@localhost:5432/desafio?sslmode=disable"
 
 func main() {
 	logger := logrus.New()
 	logger.SetFormatter(&logrus.TextFormatter{TimestampFormat: time.RFC3339})
 	lentry := logrus.NewEntry(logger)
 
-	dbAccount, err := os.OpenFile(dbFileNameAccount, os.O_RDWR|os.O_CREATE, 0666)
-
-	if err != nil {
-		lentry.Fatal("problem opening %s %v", dbFileNameAccount, err)
-	}
-
-	dbToken, err := os.OpenFile(dbFileNameToken, os.O_RDWR|os.O_CREATE, 0666)
-
+	err := postgre.RunMigrations(dbUrl)
 	if err != nil {
-		lentry.Fatal("problem opening %s %v", dbFileNameToken, err)
+		lentry.Fatal(os.Stderr, "failed to run migrations: %v\n", err)
```

Comment:
three points here:
- I guess u intended to use `Fatalf` instead of `Fatal`
- what's the purpose of this os.Stderr?
- if you're using fatal log its the os.Exit after it is not needed.